### PR TITLE
add styles to summary appendix terms microbiology

### PR DESCRIPTION
**Pull request by ajluk**
Issue: #2054

### CODE REVIEW

File: styles/output/microbiology-pdf.css [hunk 1]
```diff
@@ -865,17 +865,18 @@ nav#toc > .os-toc-title {
...
+[data-type='chapter'] > .os-summary-container a:first-of-type > h3 {
   font-family: Muli;
-  font-weight: bold;
   font-size: 1.44rem;
-  line-height: 1.5rem;
-  margin-bottom: 0; }
+  line-height: 1.5rem; }
 
-[data-type='chapter'] > .os-summary-container .summary > ul {
+[data-type='chapter'] > .os-summary-container .section-summary > ul {
```
Comment (wilkus27) [inline styles/output/microbiology-pdf.css]: There is something wrong here. In Microbio there is no tag with class `.section-summary`. It should stay `.summary`.

File: styles/output/microbiology-pdf.css [hunk 2]
```diff
@@ -865,17 +865,18 @@ nav#toc > .os-toc-title {
...
   margin-left: 24px; }
 
-[data-type='chapter'] > .os-summary-container .summary > ol {
+[data-type='chapter'] > .os-summary-container section > ul {
+  margin-left: 24px; }
+
+[data-type='chapter'] > .os-summary-container .section-summary > ol {
```
Comment (wilkus27) [inline styles/output/microbiology-pdf.css]: Same here

File: styles/books/microbiology/book.scss [hunk 3]
```diff
@@ -142,6 +151,8 @@
 @include use('FillInTheBlank','carnival:::EocCompChapExercises');
 @include use('Matching','carnival:::EocCompChapExercises');
 @include use('TrueFalse','carnival:::EocCompChapExercises');
+@include use('Appendix', 'carnival:::AppendixShape');
```
Comment (wilkus27) [inline styles/books/microbiology/book.scss]: This include is a duplicate of line 146.

File: styles/output/microbiology-pdf.css (hunk 1 above)
Comment (ajluk) [reply styles/output/microbiology-pdf.css]: Natalia, thanks for catching it up! I don't know why I thought Microbiology and College Physics are using the same shape and applied it. Changes are ready for your review.

**Comment by brittweinstein** [comment]
@ajluk we should be good to merge!

**Pull request merged**